CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ releases, in reverse chronological order.
 v2.2.0
 ------
 
+* Added support for filtering by priority
```

Comment:
Please also mention that we're dropping `--urgent` (eg: `"replace --urgent with --priority"`).